### ISSUE TITLE
Add Basic Accessibility Features

### ISSUE THREAD

**Issue by nerdyman** [nerdyman/react-image-turntable#1]
- [x] Element should be focusable and support keyboard navigation
- [x] Each image should still append the current index to its alt tag
- [x] Determine if there's an accessible (screen reader friendly) way to render the turntable
